### PR TITLE
[docs] Changelog v20

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
//cc @DevExpress/testcafe-docs @miherlosev 
Review only _CHANGELOG.md_, _assertions/README.md_ was added to another pull request

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,79 @@
...
+```js
+test
+    .meta('testID', 't-0005')
+    .meta({ severity: 'critical', testedAPIVersion: '1.0' });
+    ('MyTest', async t => { /* ... */});
+```
+
+To include testing metadata to reports, use [custom reporter methods](https://devexpress.github.io/testcafe/documentation/extending-testcafe/reporter-plugin/reporter-methods.html).
+
+#### :gear: Passing a regular promise to `t.expect` ([#2207](https://github.com/DevExpress/testcafe/issues/2207))
```
Comment (AndreyBelym) [inline CHANGELOG.md]: IMHO, header should say the opposite, like
```Passing a regular promise to t.expect is deprecated now```

Previously you could pass a regular promise without any problem. Now we throw an error in this case, and you should set a special flag to get previous behavior.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,79 @@
...
+```
+
+```js
+test
+    .meta('testID', 't-0005')
+    .meta({ severity: 'critical', testedAPIVersion: '1.0' });
+    ('MyTest', async t => { /* ... */});
+```
+
+To include testing metadata to reports, use [custom reporter methods](https://devexpress.github.io/testcafe/documentation/extending-testcafe/reporter-plugin/reporter-methods.html).
```
Comment (miherlosev) [inline CHANGELOG.md]: Add this clarification:
Meta information defined in fixture or test will not affect the build-in reporters (spec, list and etc. from https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/reporters.html).
It's actual only for custom reporters and to include  testing metadata....

Comment (miherlosev) [reply CHANGELOG.md]: After discussing we decide to move this clarification to `meta` help topic.

Comment (MargaritaLoseva) [reply CHANGELOG.md]: I'll add this info to the topic about meta, not to Changelog.

**Comment by MargaritaLoseva** [comment]
@dirk-pieterse please review

**Comment by MargaritaLoseva** [comment]
@testcafe-build-bot \retest

**Pull request merged**